tools/chpldef/test/TestClient.cpp

```diff
@@ -96,8 +96,24 @@ static std::string mentionSymbol(const chpl::uast::AstNode* ast) {
 }
 
 static Location
-locationFromAst(chpl::Context* chapel, const chpl::uast::AstNode* ast) {
-  CHPL_ASSERT(ast);
+sourceLocationFromAst(chpl::Context* chapel, const chpl::uast::AstNode* ast) {
```

Comment:
"Source" is a misleading name IMO -- `mentionLocationFromAst` would make it clear that you're finding the thing-that-mentions-an-ID

Reply:
I've gone ahead and renamed these to `mentionSourceLocation` and `mentionTargetLocation` - they're so named because they match the field names in the `Mention` struct.